.gitignore

```diff
@@ -1,13 +1,15 @@
+# Exclude common editor config
+.vscode/
 *.DS_Store
 .idea
 .settings
+# Exclude results of the build process
 idea/*
```

Comment:
```suggestion
idea/*
# Exclude results of the build process
```

`idea/` is also a non-build exclusion.

Reply:
Sounds good. And as you can see, I decided all engineering is compromise. It's good to know the intended use of `.gitignore`. But, I think there's merit to preventing a common error if someone does `git add --a`.

Reply:
I'll apply that small change and merge this now. Thanks!

Reply:
> I think there's merit to preventing a common error if someone does `git add --a`.

Yeah... I'm of two minds, really. I _do_ think that, to an extent, but the problem is that as soon as you start down that road, you end up with a file full of literally dozens of different software packages' particular ephemera. It's impossible to cover all the bases.

Plus, PRs with random spurious files in them _should_ be caught and fixed in review. If they're not, then that makes me think that either
1. We just aren't bothering to review code before merging
2. PRs being submitted are too big/sprawling for others to review them

I know there's _defintely_ some of the second, I've been guilty of it myself. The first, I have no idea... I'm personally reluctant to merge my own code without a second pair of eyes unless it's a REALLY simple, unambiguous change... and that probably explains why I have like 25 open PRs across the three repos. :confounded: 

All that being said, though, I bet there's a bot we could install to patrol for newly-added files, and call them out in the PR comments. It _should_ be rare for a PR to introduce completely new files, so that would probably do a lot to prevent accidental adds. Hmm... to the Marketplace!